token/cli/src/main.rs

```diff
@@ -1033,6 +1037,18 @@ async fn command_transfer(
                 &recipient_owner,
                 &sender_owner,
                 transfer_balance,
+                maybe_fee,
+                &bulk_signers,
+            )
+            .await?
+    } else if let Some(fee) = maybe_fee {
+        token
+            .transfer_with_fee(
+                &sender,
```

Comment:
im not a fan of the fee being a fallback here, if the user passes `--fund-recipient --expected-fee 1` and the fee is 1000000, the transfer will go through. easy solution is to make the flags mutually exclude each other, more annoying solution is to use `calculate_fee` to sanity check they arent being rugged